### PR TITLE
[G7T5-8] Create Job-Skill API

### PR DESCRIPTION
**Pull request by deadline2359**
# Description

Created an API for Job-Skill table. So, we can read from the table for learners to see skills for selected jobs.

Fixes # (issue)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Not applicable, skills table has not been merged.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: backend/app/api/api_v1/endpoints/job_skill.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+def get_job_skill_by_id(
+    job_id: int,
+    skill_id: int,
+    db: Session = Depends(deps.get_db),
+) -> Any:
+    """
+    Get a specific job and skill by their ids.
+    """
+    job_skill = crud.job_skill.get(db, job_id=job_id, skill_id=skill_id)
+    return job_skill
```
Comment (theianchia) [inline backend/app/api/api_v1/endpoints/job_skill.py:35]: Should we be trying to get all `skill_id` based on a `job_id` instead

File: backend/app/api/api_v1/api.py [hunk 2]
```diff
@@ -17,3 +17,5 @@ def health_check():
 api_router.include_router(
     registration.router, prefix="/registration", tags=["registration"]
 )
+# api_router.include_router(job.router, prefix="/job", tags=["job"])
+api_router.include_router(job_skill.router, prefix="/job_skill", tags=["job_skill"])
```
Comment (theianchia) [inline backend/app/api/api_v1/api.py:21]: EOF line

File: backend/alembic/versions/0004_create_job_skill.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Revision ID: 0004
+Revises: 0003
+Create Date: 2022-09-21 00:00:00
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "0003"
+down_revision = "0002"
```
Comment (jonathantan1425) [inline backend/alembic/versions/0004_create_job_skill.py:12]: Versions mismatched against docstring

File: backend/app/api/api_v1/api.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 from fastapi import APIRouter
 
-from app.api.api_v1.endpoints import course, registration, role, staff  # login,
+from app.api.api_v1.endpoints import course, registration, role, staff, job_skill #, job  # login,
```
Comment (jonathantan1425) [inline backend/app/api/api_v1/api.py:3]: Double `#` might not make sense here

**Review by jonathantan1425 — COMMENTED** [review]
Missing unit tests. What's the reason to remove `Job` here?

**Pull request closed without merging**